healvis/data/configs/obsparam_test.yaml

```diff
@@ -13,9 +13,8 @@ telescope:
   telescope_location: (-30.72152777777791, 21.428305555555557, 1073.0000000093132)  # LatLonAlt location
   telescope_name: HERA
 freq:
-  Nfreqs: 3
-  start_freq: 100000000.0  # Hz
-  bandwidth:  100000000.0  # Hz
+  freq_array: [1.00e+08, 1.05e+08, 1.10e+08, 1.15e+08, 1.20e+08,
```

Comment:
I had it in `run_simulation_partial_freq` that you can feed either 
1) `Nfreqs`, `start_freq` and `bandwidth` or
2) `freq_array` and if the latter was fed it superceded the former. can we keep this here?

Reply:
and of course we should make that clear via comments

Reply:
I've put it back as `Nfreqs`, `start_freq`, and `bandwidth`. It's still consistent with the freqs in the gsm_nside32 test file.